tests/build-timeout.unit.test.ts

```
@@ -0,0 +1,44 @@
+import { describe, expect, it } from 'vitest'
+import {
+  DEFAULT_BUILD_TIMEOUT_SECONDS,
+  MAX_BUILD_TIMEOUT_SECONDS,
+  MIN_BUILD_TIMEOUT_SECONDS,
+  calculateBuildRuntimeSeconds,
```

ESLint

[error] Expected "calculateBuildRuntimeSeconds" to come before "MIN_BUILD_TIMEOUT_SECONDS".

(perfectionist/sort-named-imports)